projects/components/src/overlay/modal/modal-overlay.component.scss

```diff
@@ -0,0 +1,48 @@
+@import 'color-palette';
+@import 'font';
+
+.modal-overlay {
+  box-shadow: -1px -1px 8px rgba(0, 0, 0, 0.08), -3px 1px 24px rgba(0, 0, 0, 0.12);
```

Comment:
Is this box shadow different than the ones in interactions.scss?

Reply:
Yes it is different. And this is temporary as well as I would by applying some more style in a follow up PR.